packages/web-components/src/appointment-planner/server/serverless.yml

```diff
@@ -0,0 +1,90 @@
+service: web-components.appointment-planner
+plugins:
+  - serverless-webpack
+  - serverless-offline
+  - serverless-plugin-ifelse
+
+custom:
+  # relative to web-components package root
+  env: ${file(./config.json)}
+  webpack:
+    webpackConfig: 'webpack.config.js'
+    includeModules: true
+    packager: 'yarn'
+    packagerOptions:
+      noFrozenLockFile: false
+    excludeFiles: src/**/*.test.ts
+    keepOutputDirectory: true
+  allowedHeaders:
+    - Content-Type
+    - X-Amz-Date
+    - Authorization
+    - X-Api-Key
+    - X-Amz-Security-Token
+    - X-Amz-User-Agent
+    - reapit-customer
+
+provider:
+  name: aws
+  runtime: nodejs10.x
+  stage: ${opt:stage, 'dev'}
+  region: eu-west-2
+  apiGateway:
+    restApiId: ${self:custom.env.GATEWAY_REST_API_ID}
+    restApiRootResourceId: ${self:custom.env.GATEWAY_REST_API_ROOT_RESOURCE_ID}
+  deploymentBucket:
+    name: web-components.appointment-planner.${self:provider.stage}
```

Comment:
`web-components-appointment-planner.${self:provider.stage}`
instead

Reply:
And you need to create a bucket named `web-components-appointment-planner-dev` in AWS S3 if it hasn't existed yet